samples/snippets/cloud-client/src/main/java/recaptcha/ListSiteKeys.java

```diff
@@ -48,10 +49,12 @@ public static void listSiteKeys(String projectID) throws IOException {
       ListKeysRequest listKeysRequest =
           ListKeysRequest.newBuilder().setParent(ProjectName.of(projectID).toString()).build();
 
+      ListKeysPagedResponse response = client.listKeys(listKeysRequest);
       System.out.println("Listing reCAPTCHA site keys: ");
-      for (Key key : client.listKeys(listKeysRequest).iterateAll()) {
+      for (Key key : response.iterateAll()) {
         System.out.println(key.getName());
       }
+      return response;
```

Comment:
No need to return the response, keep void. https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md#method-structure

Reply:
This response is being used in the MigrateKey file. Returning void would mean adding this piece of code to the MigrateKey file (which would be redundant).
Is it okay to treat cases like this as an exception?

Reply:
Should avoid returning any value wherever possible, but otherwise fine